### DISCUSSION TITLE
Proposta grafica sezione inserimento argomento/compiti docente

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#764] [Idee e proposte]
<img width="1231" alt="Screenshot 2024-10-12 alle 15 17 27" src="https://github.com/user-attachments/assets/c0d5fbe8-b148-4a7a-8167-6905b98ca55d">

**Comment by federicabianchi08** [comment]
È molto funzionale , hai ragione.

**Comment by Davide031** [comment]
Bell' idea soprattutto per lo spazio che è disponibile solo ai docenti (_es lettura fino a pag 10_) come hai fatto a farlo?

**Reply by sempliciniluigi89** [reply]
Con word ,ma non ho la minima idea di come si possa rendere effettivo all'interno del registro .
